quarkus/config-api/src/main/java/org/keycloak/config/HotRodOptions.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.config;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HotRodOptions {
```

Comment:
For now, we are keeping all storage options within the `Storage` configuration namespace. That means moving all the options here to the `StorageOptions`.

Their name should follow something like `storage-hotrod-<name>`.

We not yet set in stone how the configuration is going to look like, that is why we putting everything into a single namespace so that we can track all options we need to consider from a single place.